src/dsp_tools/commands/ingest_xmlupload/upload_xml.py

```diff
@@ -0,0 +1,70 @@
+from __future__ import annotations
+
+from pathlib import Path
+
+from lxml import etree
+
+from dsp_tools.commands.ingest_xmlupload.apply_ingest_id import (
+    get_mapping_dict_from_file,
+    replace_filepath_with_sipi_id,
+)
+from dsp_tools.commands.xmlupload.upload_config import UploadConfig
+from dsp_tools.commands.xmlupload.xmlupload import xmlupload
+from dsp_tools.models.exceptions import InputError
+from dsp_tools.utils.create_logger import get_logger
+from dsp_tools.utils.xml_utils import remove_comments_from_element_tree
+
+logger = get_logger(__name__)
+
+
+def ingest_xmlupload(
+    xml_file: Path,
+    user: str,
+    password: str,
+    dsp_url: str,
+    sipi_url: str,
+) -> None:
+    """
+    This function reads an XML file
+    and imports the data described in it onto the DSP server,
+    using the ingest XML upload method.
```

Comment:
```suggestion
```

Reply:
When I suggested to delete this line, it was not clear to me what the "ingest XML upload method" is. But probably that's the term you use for sending a request with the header "X-Asset-Ingested". Then it makes sense.